language/tools/move-unit-test/src/test_runner.rs

```diff
@@ -279,6 +279,13 @@ impl SharedTestingConfig {
                         pass(function_name);
                         stats.test_success();
                     }
+                    // Expected the test to abort and it aborted with internal error
+                    (Some(ExpectedFailure::Expected), None)
```

Comment:
I'm thinking we probably want to add in a different error type to represent these types of errors. In my mind when I was implementing this, it was a way of saying "this needs to abort, but I don't care with what abort code" which is different from  "this should fail with something other than an abort code."

What would you think about adding in a new failure type for this? e.g., 
```
#[expected_failure(abort)] // this should abort but I don't care about the code
#[expected_failure(abort = <code>)] // this should abort with <code>
#[expected_failure(internal)] // this should fail with an internal error (i.e., any error other than abort or OUT_OF_GAS)
```

It's fine to keep this as-is for now, but could you possibly file an issue for this, and you or I can add it in after this PR lands.

Reply:
I like the `#[expected_failure(internal)]` failure type for this. I'll file an issue.